### PR TITLE
Helm chart for central fluentd log aggregator

### PR DESCRIPTION
**Pull request by leahnp**

### CODE REVIEW

File: central-fluentd/.helmignore [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
```
Comment (coffeepac) [inline central-fluentd/.helmignore]: should add .DS_Store to this file to exclude that file

Comment (leahnp) [inline central-fluentd/.helmignore]: Changes made!

Comment (coffeepac) [inline central-fluentd/.helmignore]: there is another .DS_Store file in the templates directory.  please remove that one as well.

Comment (leahnp) [inline central-fluentd/.helmignore]: Ok, did that.

**Comment by coffeepac** [comment]
LGTM

**Pull request merged**